go.mod

```diff
@@ -1,31 +1,95 @@
 module github.com/ONSdigital/dp-observation-importer
 
-go 1.16
+go 1.17
 
 require (
-	github.com/ONSdigital/dp-api-clients-go/v2 v2.0.6-beta
-	github.com/ONSdigital/dp-component-test v0.2.0
-	github.com/ONSdigital/dp-graph/v2 v2.7.3
-	github.com/ONSdigital/dp-healthcheck v1.1.0
-	github.com/ONSdigital/dp-kafka/v2 v2.1.2
+	github.com/ONSdigital/dp-api-clients-go/v2 v2.3.0
+	github.com/ONSdigital/dp-component-test v0.6.0
+	github.com/ONSdigital/dp-graph/v2 v2.15.0
+	github.com/ONSdigital/dp-healthcheck v1.1.3
+	github.com/ONSdigital/dp-kafka/v2 v2.4.1
 	github.com/ONSdigital/dp-net v1.2.0
 	github.com/ONSdigital/dp-reporter-client v1.1.0
-	github.com/ONSdigital/go-ns v0.0.0-20200902154605-290c8b5ba5eb
+	github.com/ONSdigital/go-ns v0.0.0-20210916104633-ac1c1c52327e
 	github.com/ONSdigital/log.go/v2 v2.0.9
-	github.com/cucumber/godog v0.10.0
-	github.com/gofrs/uuid v3.3.0+incompatible // indirect
+	github.com/cucumber/godog v0.12.1
+	github.com/gofrs/uuid v4.0.0+incompatible // indirect
 	github.com/gogo/protobuf v1.3.2 // indirect
 	github.com/golang/mock v1.6.0
 	github.com/gorilla/mux v1.8.0
 	github.com/gorilla/websocket v1.4.2 // indirect
-	github.com/hashicorp/go-memdb v1.3.0 // indirect
+	github.com/hashicorp/go-memdb v1.3.2 // indirect
 	github.com/johnnadratowski/golang-neo4j-bolt-driver v0.0.0-20200323142034-807201386efa
 	github.com/kelseyhightower/envconfig v1.4.0
-	github.com/maxcnunes/httpfake v1.2.2
+	github.com/maxcnunes/httpfake v1.2.3
 	github.com/patrickmn/go-cache v2.1.0+incompatible
 	github.com/rdumont/assistdog v0.0.0-20201106100018-168b06230d14
 	github.com/smartystreets/goconvey v1.6.4
 	github.com/stretchr/testify v1.7.0
 	gopkg.in/check.v1 v1.0.0-20201130134442-10cb98267c6c // indirect
-	gopkg.in/yaml.v3 v3.0.0-20200615113413-eeeca48fe776 // indirect
+	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b // indirect
```

Comment:
Should we keep all indirect dependencies in the other require group?

Reply:
I just do what `go mod tidy` does